<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="model.syncManager.ItemSyncManagerImplTest" tests="4" skipped="0" failures="0" errors="0" timestamp="2017-10-20T17:48:03" hostname="raphael-laptop" time="0.404">
  <properties/>
  <testcase name="testA" classname="model.syncManager.ItemSyncManagerImplTest" time="0.103"/>
  <testcase name="testB" classname="model.syncManager.ItemSyncManagerImplTest" time="0.114"/>
  <testcase name="testC" classname="model.syncManager.ItemSyncManagerImplTest" time="0.039"/>
  <testcase name="testD" classname="model.syncManager.ItemSyncManagerImplTest" time="0.147"/>
  <system-out><![CDATA[javax.validation.ConstraintViolationException: Bean Validation constraint(s) violated while executing Automatic Bean Validation on callback event:'prePersist'. Please refer to embedded ConstraintViolations for details.
]]></system-out>
  <system-err><![CDATA[]]></system-err>
</testsuite>
